### DISCUSSION TITLE
Filtering nodes by edge count

### DISCUSSION THREAD

**Discussion by liking4viking** [gephi/gephi#2618] [Q&A]
Dear all, 

Currently, I am working on a narrative network of a historical source. I was wondering what is the best way to filter nodes (here individuals) who are mentioned in the text by edge count (here  relationships in the text)? I am particularly interested with people who have zero edges but are still considered fairly important individuals so that I can return to the document and study these areas in more detail.

I have looked at the filters but it seems to want to sort the edges by weight but, I just want to see if I can extract how many individuals have 0 or 1 edge in the network. 

Any help would be appreciated. Please let me know if you need additional information. 

All the best.

**Comment by mbastian** [comment]
Hi @liking4viking, isn't what you'r looking for a simple Degree Filter? The degree is the number of edges a node has. You can find htis filter in the Topology category.
